dephell/converters/setuppy.py

```diff
@@ -12,6 +12,7 @@
 from dephell_specifier import RangeSpecifier
 from packaging.requirements import Requirement
 from setuptools.dist import Distribution
+from sortedcontainers import SortedDict
```

Comment:
Why not OrderedDict? :thinking:

Reply:
Well, I see a lot of lists and dicts which should be sorted, and using sortedcontainers makes sorting the default, while OrderedDict means `sorted(..)` needs to always be used.

I can do OrderedDict if you would prefer.

And it makes sense for me to add `package_data` too; I only held off on it as I didnt have time before to find a sample to test it on.

Reply:
Yes, stdlib is preferrable. DepHell has a bunch of necessary dependencies, and it makes some users worry about. So, I'm trying to keep the dependencies list for dephell as small as possible. BTW, I'm going to drop `flatdict` as well. If you can do it by stdlib and a few more codelines, please, do.

BTW, `project.entrypoints` should be already sorted, and probably you don't need to sort lists, only the dict itself.

Reply:
BTW, as I mentioned in #265, `json.dumps` with `sort_keys=True` should work because python and JSON have the same syntax for dict, list, and string ;)

Reply:
@orsinium , `json.dumps` emits a string, which is then `{!r}`, so the result is

```diff
-    entry_points={'console_scripts': ['dephell = dephell.cli:entrypoint']},
+    entry_points='{"console_scripts": ["dephell = dephell.cli:entrypoint"]}',
```

Also as you can see, the JSON dumper is using double quotes when previously single quotes were used.

Worked around those.  Hopefully this suits you.